src/index.html

```diff
@@ -7,8 +7,32 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;800&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="header__logo"><img src="./images/logo.png" alt="logo__MOYO"></a>
```

Comment:
```suggestion
      <a href="#" class="header__logo">
        <img src="./images/logo.png" alt="logo__MOYO">
      </a>
```
Consider fixing indents according to the code style guide